index.html

```diff
@@ -5,6 +5,7 @@
   <head>
     <title>Cocktail Builder</title>
     <link rel='stylesheet' href='styles/normalize.css' type='text/css'>
+    <link rel='stylesheet' href='vendor/styles/bootstrap/css/bootstrap.min.css' type='text/css'>
```

Comment:
You should use bootstrap from a CDN, not a local copy. Will be faster.  http://getbootstrap.com/getting-started/